gimme_free_epic_shit.js

```diff
@@ -1,50 +1,54 @@
 const { Launcher } = require(`epicgames-client`);
 const { email, password } = require(`${__dirname}/config.json`);
-const epic = new Launcher({
+const defaultPageSize = 100;
+const epicLauncher = new Launcher({
```

Comment:
Still a bit undecided on this. I don't really like the whole launcher name, as it's not really a launcher. Something like `EpicGames` or `EpicClient` or `client` would fit more I think.

Reply:
changed to epicClient